tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+
+from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
+from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.test import BluechiTest
+
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+    multi_user_string = "multi-user.target"
+    graphical_string = "graphical.target"
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
+    assert systemd_result in bc_result
```

Comment:
Why using `in` ? Shouldn't the results be equals?

Reply:
The result that I get in bc_result is `The default target of this node is: graphical.target`.
That is why I used `in`

Reply:
See my other comment, that's your message, but to align with systemctl you should print only the returned target

Reply:
OK